commands/error.go

```diff
@@ -437,6 +439,28 @@ func IsUnspecifiedAPIError(err error) bool {
 	return microerror.Cause(err) == unspecifiedAPIError
 }
 
+// noUpgradeAvailableError means that the user wanted to start an upgrade, but
+// there is no newer version available for the given cluster
+var noUpgradeAvailableError = &microerror.Error{
+	Kind: "noUpgradeAvailableError",
+	Desc: "no upgrade available for the current version",
```

Comment:
IMO these are supposed to be sentences. See https://godoc.org/github.com/giantswarm/microerror#Error.